src/apps/reports/src/app/views/ActivityLog/components/ActionTimelineItem/TimelineItem.js

```diff
@@ -63,9 +63,16 @@ export const TimelineItem = (props) => {
         return "Unpublished";
       case 6:
         const [publishAt] = props.action?.meta?.message.split(" ").slice(-1);
-        return `Scheduled to Publish on ${moment(publishAt).format(
-          "MMMM DD [at] hh:mm A"
-        )}`;
+        const happenedAt = props.action?.happenedAt;
+        const format = "MMMM DD [at] hh:mm A";
+        const convertedDate = moment(
+          new Date(publishAt),
+          format,
```

Comment:
In the documentation, they are passing format and third parameter to make it stricter. If you try to use only isValid on a date string without formatting and adding true as 3rd argument it will have an issue. Try passing "abcd-1" and check isValid it will return true. In the reference i send there are some example regarding that

Reply:
If this is not ideal approach , I can make this simple by just using moment(dateString).isValid(). Would this is you prefer approach?